javascript/integration-tests/FailTransfer.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('FailTransfer', (): void => {
 
     it('fee is min 0.01 CTC', async () => {
         const transferId = createFundingTransferId('Ethereum', '0xffffffffffffffffffffffffffffffffffffffff');
-        const cause = api.createType('PalletCreditcoinOcwErrorsVerificationFailureCause', 'TransferFailed');
+        const cause = api.createType('PalletCreditcoinOcwErrorsVerificationFailureCause', 'TaskFailed');
```

Comment:
Rebase onto `dev` and fix in `integration-tests/src/test/fail-transfer.test.ts`.